apollo/src/jsMain/kotlin/io/iohk/atala/prism/apollo/securerandom/SecureRandom.kt

```diff
@@ -19,13 +20,21 @@ actual class SecureRandom actual constructor(
      */
     override fun nextBytes(size: Int): ByteArray {
         val arr = Uint8Array(size)
-        return crypto.getRandomValues(arr).buffer.toByteArray()
+        return if (isNode) {
+            _require("crypto").randomFillSync(arr)
+        } else {
+            global.crypto.getRandomValues(arr)
+        }
```

Comment:
Because:
- The `crypto` module is not the same for the `Node.js` and `Browser` environments. Both have different usage interfaces.
- If the `Node.js` module is used in `Browser`, it fails with the module not found, and the same happens when reversed.

That's why I implemented them like that.

For reference, check #135